lib/src/main/kotlin/com/swmansion/starknet/account/StandardAccount.kt

```diff
@@ -87,15 +85,14 @@ class StandardAccount @JvmOverloads constructor(
         nonce: Felt,
         forFeeEstimate: Boolean,
     ): DeployAccountTransactionV1Payload {
-        val signVersion = if (forFeeEstimate) TransactionVersion.V1_QUERY else TransactionVersion.V1
```

Comment:
Instead of passing `version`, we can pass `forFeeEstimate` flag to the class constructor.